src/machine.rs

```diff
@@ -253,17 +251,16 @@ impl OlmMachine {
     ) -> napi::Result<
         Vec<
             // We could be tempted to use `requests::OutgoingRequests` as its
-            // a type alias for this giant `Either7`. But `napi` won't unfold
+            // a type alias for this giant `Either6`. But `napi` won't unfold
             // it properly into a valid TypeScript definition, so…  let's
             // copy-paste :-(.
-            Either7<
+            Either6<
```

Comment:
Final nitpick, the function comment should remove its mention of `KeysBackupRequest` now.